user-org-es-util/src/main/java/org/sunbird/dto/SearchDtoMapper.java

```diff
@@ -0,0 +1,166 @@
+package org.sunbird.dto;
+
+import org.sunbird.dto.SearchDTO;
+
+import java.math.BigInteger;
+import java.util.*;
+
+/**
+ * This class will be used to map the SearchDto
+ *
+ * @author anmolgupta
+ */
+
+
+public class SearchDtoMapper {
+
+
+    /**
+     * this method will create SearchDto pojo object.
+     * @param searchQueryMap
+     * @return
+     */
+    public static SearchDTO createSearchDto(Map<String, Object> searchQueryMap) {
+        SearchDTO search = new SearchDTO();
+        search = searchForQueryInRequest(searchQueryMap, search);
```

Comment:
If you create a separate class, there is no need to pass the second param (search)...You can simply do a series of setters and finally say, build() to return you a SearchDTO instance.

Also, rename searchForFieldsInRequest -> setFields()
searchForSortByInRequest -> setSortBy() and alike. Keep names simple, they are setting, not searching...